docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md

```diff
@@ -1,58 +1,137 @@
-# Running a Kubernetes Cluster on vSphere with kubeadm
-
+# Deploying a Kubernetes Cluster on vSphere with CSI and CPI
+The purpose of this guide is to provide the reader with step by step instructions on how to deploy Kubernetes on vSphere infrastructure. The instructions use kubeadm, a tool built to provide best-practice “fast paths” for creating Kubernetes clusters. The reader will also learn how to deploy the Container Storage Interface and Cloud Provider Interface plugins for vSphere specific operations.
 ## Prerequisites
+This section will cover the prerequisites that need to be in place before attempting the deployment.
+## vSphere requirements
+vSphere 6.7U3 is a prerequisite for using CSI and CPI at the time of writing. This may change going forward, and the documentation will be updated to reflect any changes in this support statement.
+## Recommended Guest Operating System
+VMware recommends that you create a virtual machine template using Guest OS Ubuntu 18.04.1 LTS (Bionic Beaver) 64-bit PC (AMD64) Server. Check it out on VMware PartnerWeb here. This template is cloned to act as base images for your Kubernetes cluster. For instructions on how to do this, please refer to the guidance provided in this blog post by Myles Gray of VMware. Ensure that SSH access is enabled on all nodes. This must be done in order to run commands on both the Kubernetes master and worker nodes in this guide.
+## Virtual Machine Hardware requirements
+Virtual Machine Hardware must be version 15 or higher. For Virtual Machine CPU and Memory requirements, size adequately based on workload requirements. 
+VMware also recommend that virtual machines use the VMware Paravirtual SCSI controller for Primary Disk on the Node VM.
```

Comment:
I think this is the default now? Maybe just make a note that it should be that way OOTB but good to check.

Reply:
Done.